test/Windows.hs

```diff
@@ -231,13 +239,19 @@ parseAbsFileSpec =
      failing "\\"
      failing "\\\\"
      failing "\\\\\\foo\\\\bar\\\\mu\\"
-     -- succeeding "\\..." (Path "\\...") FIXME
-     -- succeeding "\\foo.txt" (Path "\\foo.txt") FIXME
-     -- succeeding "C:\\\\\\foo\\\\bar\\\\\\\\mu.txt" (Path "C:\\foo\\bar\\mu.txt") FIXME
-     -- succeeding "C:\\\\\\foo\\\\bar\\.\\\\mu.txt" (Path "C:\\foo\\bar\\mu.txt") FIXME
+     failing "\\..."
+     failing "\\foo.txt"
+     succeeding "C:\\\\\\foo\\\\bar\\\\\\\\mu.txt" (Path "C:\\foo\\bar\\mu.txt")
+     succeeding "C:\\\\\\foo\\\\bar\\.\\\\mu.txt" (Path "C:\\foo\\bar\\mu.txt")
+     succeeding "\\\\unchost\\share\\\\file.txt" (Path "\\\\unchost\\share\\file.txt")
+     succeeding "\\/unchost\\share\\\\file.txt" (Path "\\\\unchost\\share\\file.txt")
+     succeeding "\\\\unchost\\share\\.\\folder\\\\\\file.txt" (Path "\\\\unchost\\share\\folder\\file.txt")
+     succeeding "\\\\?\\C:\\file.txt" (Path "\\\\?\\C:\\file.txt")
+     succeeding "/\\?\\C:\\file.txt" (Path "\\\\?\\C:\\file.txt")
+     succeeding "\\\\?\\C:\\\\\\folder\\.\\\\file.txt" (Path "\\\\?\\C:\\folder\\file.txt")
```

Comment:
`filepath` says something about some of these not being relative paths: http://hackage.haskell.org/package/filepath-1.4.2.1/docs/System-FilePath-Windows.html#v:isRelative

Can you check?

If these in fact _are_ valid relative paths, can we correctly append them with `</>`?

Reply:
That's under `parseAbsFileSpec`, so we're testing that they're valid absolute paths on these lines.

Under `parseRelFileSpec`, there are tests that these patterns fail.

Reply:
> That's under parseAbsFileSpec, so we're testing that they're valid absolute paths on these lines.

Oops, my bad.

> I'll look at adding tests for </>.

:+1:

Reply:
Tests added